app/src/main/java/org/covidwatch/android/presentation/SplashFragment.kt

```diff
@@ -14,6 +15,17 @@ class SplashFragment : Fragment() {
     private var _binding: FragmentSplashBinding? = null
     private val binding get() = _binding!!
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val onBackPressed: OnBackPressedCallback = object : OnBackPressedCallback(true) {
```

Comment:
What does this do?

Reply:
this is for the case where user is in splash screen, and he hits back button (we need to close the app in this case). without this we'll go back to HomeFragment as this is our new starting destination in nav-graph :)